src/EnergyPlus/DataGlobalConstants.hh

```diff
@@ -571,7 +571,10 @@ namespace Constant {
     Real64 constexpr OneFifth = 1.0 / 5.0;   // 1/5 in highest precision
     Real64 constexpr OneSixth = 1.0 / 6.0;   // 1/6 in highest precision
     Real64 constexpr FourFifths = 4.0 / 5.0; // 4/5 in highest precision
+    Real64 constexpr OneThousandth = 1.0e-3; // Used as a tolerance in various places
+    Real64 constexpr OneMillionth = 1.0e-6;  // Used as a tolerance in various places
 
+    Real64 constexpr DistTooSmall(1.e-4);      // Geometric tolerance
```

Comment:
Added some new constants that are used as tolerances in various places in surface geometry and shadowing calculations. `DistToolSmall` was moved up from a local declaration in Vectors.cc - decided to keep the same name.

Reply:
Does DistTooSmall represent a unit or is it dimensionless?

Reply:
> Does DistTooSmall represent a unit or is it dimensionless?

Good questions. After some digging, it's in meters. And after some discussion with @JasonGlazer about other geometric tolerances, I'm going to make some other changes. How do you feel about names like this?
`Real64 constexpr 1centimeter = 0.01 // Distance tolerance [m]`

Reply:
Well, I mean that exact name won't compile since it starts with a numeric literal :)    But I think `oneCentimeter` is fine.  Or you could just call it oneHundredth and allow it to be used for multiple units.  As long as it's pretty obvious I'm not too picky.